Vic2ToHoI4/Source/OutHoi4/OutFlags.cpp

```diff
@@ -358,6 +366,81 @@ std::tuple<uint8_t, uint8_t, uint8_t> HoI4::getDominionFlagBaseColor(std::string
 }
 
 
+std::optional<tga_image*> HoI4::createUnrecognizedNationFlag(const std::string& hoi4Suffix,
+	 const std::string& vic2Suffix,
+	 const std::string& region,
+	 const Mods& vic2Mods)
+{
+	constexpr int sizeX = 93;
+	constexpr int sizeY = 64;
+
+	const auto flag = new tga_image;
+	flag->image_id_length = 0;
+	flag->color_map_type = TGA_COLOR_MAP_ABSENT;
+	flag->image_type = TGA_IMAGE_TYPE_BGR;
+	flag->color_map_origin = 0;
+	flag->color_map_length = 0;
+	flag->color_map_depth = 0;
+	flag->origin_x = 0;
+	flag->origin_y = 0;
+	flag->width = sizeX;
+	flag->height = sizeY;
+	flag->pixel_depth = 32;
+	flag->image_descriptor = 8;
+	flag->image_id = nullptr;
+	flag->color_map_data = nullptr;
+
+	flag->image_data = static_cast<uint8_t*>(malloc(sizeX * sizeY * 4));
+	if (flag->image_data == nullptr)
+	{
+		return flag;
+	}
+
+	for (unsigned int y = 0; y < sizeY; y++)
+	{
+		for (unsigned int x = 0; x < sizeX; x++)
+		{
+			const auto destIndex = (y * sizeX + x) * 4;
+
+			flag->image_data[destIndex + 0] = 128;
```

Comment:
A bit of color, per favor, some grayish, some bluish... otherwise they all look the same especially if one next to another.

Reply:
Will also be handled by #1179